ui/app/upload_image/controller.js

```diff
@@ -32,24 +38,41 @@ const handleUploadImageRoute = [
 ]
 
 /**
- * Walks through all the needed functions and checks to post an image to Slothbucket's 'classify-image' route
- * @param file {Object} - Object representing a file
+ * Posts to Slothbucket's "classify-image" route
+ * @param base64 {string} - base64 image
  * @param token {string} - JWT of authenticated user
- * @returns {Promise} - Promise representing if posting to Slothbucket was successful
+ * @returns {Promise} - Promise representing if posting to the route was successful
  */
-async function handlePostingImageToAPI(file, token) {
-    return new Promise((resolve) => {
-        if (file && token) {
-            const base64Image = validateImageAndBufferToBase64(file)
-            resolve(handleHttpPost(base64Image, token))
-        } else throw returnError.generalInvalidArgument()
+function uploadImage(base64, token) {
+    return new Promise((resolve, reject) => {
+        const options = {
+            url: 'http://localhost:8000/classify-image',
```

Comment:
For the URLs, it would be better to move it out to some config (file or environment variable). Once this is deployed, it's not a guarantee that the backend is broadcasting on localhost or port 8000